agent/hcp/telemetry_provider.go

```diff
@@ -21,7 +19,9 @@ var (
 	// internalMetricRefreshFailure is a metric to monitor refresh failures.
 	internalMetricRefreshFailure []string = []string{"hcp", "telemetry_config_provider", "refresh", "failure"}
 	// internalMetricRefreshSuccess is a metric to monitor refresh successes.
-	internalMetricRefreshSuccess []string = []string{"hcp", "telemetry_config_provider", "refresh", "success"}
+	internalMetricRefreshSuccess          []string = []string{"hcp", "telemetry_config_provider", "refresh", "success"}
+	defaultTelemetryConfigRefreshInterval          = 1 * time.Minute
+	defaultTelemetryConfigFilters                  = regexp.MustCompile(".+")
```

Comment:
@jjti had this filter as a default in his example PR, I'm not sure we want to collect metrics at the start?